charts/pega/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 name: pega
-version: "1.0.0"
+version: "0.0.1"
```

Comment:
Why is this back to `0.0.1`?  Is it dynamically set as part of the deploy process so we don't need to worry about it here?

Reply:
This might be due to the generation of chart yaml during initial refactoring not in deploy process. As part of deploy process, Chart.yaml version gets auto updated only on the gz files uploaded to bintray. The version in the source code will remain same until some one manually changes it.
Shall we change it back to 1.0.0 or 1.2.0?